CHANGELOG.md

```diff
@@ -11,182 +11,212 @@ tag versions. The Bond compiler (`gbc`) and
-  classic `.csproj` format. Any .NET Core projects that encounter the build
-  error "Duplicate BondCodegen items were included." and were explicitly
-  listing `BondCodegen` items will either need to rely on implicit codegen
-  or [disable all implicit inclusion](https://aka.ms/sdkimplicititems). To
-  set per-item metadata, use the [item update
+* **Breaking change** The Bond.CSharp and Bond.Compiler.CSharp NuGet packages
+  perform implicit codegen when the simplified .NET Core `.csproj` format is
+  used. This breaking change *does not* affect projects using the classic
+  `.csproj` format. Any .NET Core projects that encounter the build error
+  "Duplicate BondCodegen items were included." and were explicitly listing
+  `BondCodegen` items will either need to rely on implicit codegen or [disable
+  all implicit inclusion](https://aka.ms/sdkimplicititems). To set per-item
+  metadata, use the [item update
   syntax](https://docs.microsoft.com/en-us/visualstudio/msbuild/item-element-msbuild#examples).
   [Issue #636](https://github.com/Microsoft/bond/issues/636)
-* The C# attribute `Bond.Attribute` can now be applied to methods. This
-  fixes broken codegen when attributes are used on service methods.
-  [Issue #617](https://github.com/Microsoft/bond/issues/617)
+* The C# attribute `Bond.Attribute` can now be applied to methods. This fixes
+  broken codegen when attributes are used on service methods. [Issue
+  #617](https://github.com/Microsoft/bond/issues/617)
 * Bond Attributes on service methods are now present on all the client
   overloads for the methods. Previously, just the "friendly" method had the
   attributes.
 * Grpc.Core v1.12.0 is now required to use Bond-over-gRPC.
-    * This version include a number of memory leak fixes that users of Bond-over-gRPC were encountering. [Issue #810](https://github.com/Microsoft/bond/issues/810)
-    * This version include some Windows-specific performance improvements for loopback connections.
+    * This version include a number of memory leak fixes that users of
+      Bond-over-gRPC were encountering. [Issue
+      #810](https://github.com/Microsoft/bond/issues/810)
+    * This version include some Windows-specific performance improvements for
+      loopback connections.
 * `BondCodegen` items will now appear in the Visual Studio 2017+ UI in .NET
   Core projects.
-* The .NET Standard assemblies are fully strong-name signed. Previously,
-  they were inadvertently only public strong-name signed.
-* Fixed a bug in the codegen targets when using `gbc` from $PATH on macOS
-  and Linux that prevented the C# compiler from finding the generated C#
-  files.
-* *Preview*: Added preliminary support for generating types with
-  constructors with parameters for each field. This functionality will
-  change in the future and may be removed. [Pull request
+* The .NET Standard assemblies are fully strong-name signed. Previously, they
```

Comment:
Do we need to mention the signature type change? sha1->sha2

Reply:
We can, sure.